css/interface.css

```diff
@@ -24,30 +76,39 @@ body {
   padding-bottom: 120px; /* Compensate for the top part of the Studio UI */
 }
 
-#data-sources {
+#data-sources,
```

Comment:
Give these tables/containers a class instead so you don't need to repeat the CSS selectors